src/index.html

```diff
@@ -1,13 +1,413 @@
+              Range & Integrated lights
+            </h3>
+
+            <div class="details__article">
+              The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.
+            </div>
+          </div>
+
+          <div class="details__item">
+            <div class="details__grid">
+              <div class="
+                details__item--1-2
+                details__item--T-1-2
+                details__item--D-1-4
+              ">
+                <div class="details__photo-cont">
+                  <img
+                    src="images/photos/5.png"
+                    alt="Hydraulic disc brakes & Lightweight"
+                    class="details__photo"
+                  >
+                </div>
+              </div>
+
+              <div class="
+                details__item--2-2
+                details__item--T-3-6
+                details__item--D-5-12
+              ">
+                <div class="details__photo-cont">
+                  <img
+                    src="images/photos/6.png"
+                    alt="Hydraulic disc brakes & Lightweight"
+                    class="details__photo"
+                  >
+                </div>
+              </div>
+            </div>
+
+            <h3 class="details__subtitle">
+              Hydraulic disc brakes & Lightweight
+            </h3>
+
+            <div class="details__article">
+              Brakes with total stopping power the second you make contact. The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.
+            </div>
+          </div>
+
+          <div class="details__button">
+            <a href="#" class="button button__explore">
+              Explore
+            </a>
+          </div>
+        </div>
+      </section>
+
+      <section class="main__section contacts" id="contact-us">
+        <div class="container">
+          <h2 class="main__section-title contacts__title">
+            Contact us
+          </h2>
+
+          <div class="contacts__content">
+            <form action="/" class="contacts__form-column">
+              <input
+                name="name"
+                type="text"
+                placeholder="name"
+                class="contacts__field form-field"
+              >
+
+              <input
+                name="email"
+                type="email"
+                placeholder="Email"
+                class="contacts__field form-field"
+              >
+
+              <textarea
+                name="message"
+                placeholder="Message"
+                class="
+                  contacts__field
+                  contacts__field--last
+                  form-field
+                  form-field--textarea
+                "
+              >
+              </textarea>
```

Comment:
Fix formatting here, cursor position is wrong
```suggestion
              <textarea
                name="message"
                placeholder="Message"
                class="
                  contacts__field
                  contacts__field--last
                  form-field
                  form-field--textarea
                "
             ></textarea>
```
![image](https://user-images.githubusercontent.com/37161315/170648897-fec74fd5-ec57-45f9-a865-f27823ac9608.png)